osbs/build/build_request.py

```diff
@@ -481,6 +481,21 @@ def set_reactor_config_map(self):
         }
         custom['env'].append(reactor_config_map)
 
+    def set_reactor_config_yaml(self):
+        #set reactor config for workers
+        if self.spec.platforms.value is not None:
+            return
+
+        if not self.spec.reactor_config_yaml.value:
+            return
+        custom = self.template['spec']['strategy']['customStrategy']
+
+        reactor_config_yaml = {
+            'name': 'REACTOR_CONFIG',
+            'value': self.spec.reactor_config_yaml.value
```

Comment:
This is the point that `reactor_config` needs to be serialized, if not already done.

Reply:
ok